geonode_mapstore_client/client/js/routes/Home.jsx

```diff
@@ -326,6 +339,7 @@ function Home({
     );
 
     const isHeroVisible = !disableHero && inView;
+    const stickyFiltersMaxHeight = (window.innerHeight - dimensions.brandNavbarHeight - dimensions.menuIndexNodeHeight - dimensions.footerNodeHeight);
```

Comment:
you could use the prop height provided by the withResizeDetector HOC instead of the window.innerHeight